docs/source/v1/Cryosphere/simulation_data/simulation_table.rst

```diff
@@ -0,0 +1,25 @@
+**********************************
```

Comment:
This page would be better reviewed with the [preview link](https://portal.nersc.gov/cfs/e3sm/forsyth/data_docs_64_try3/html/v1/Cryosphere/simulation_data/simulation_table.html)